netket/machine/jax.py

```diff
@@ -231,6 +231,10 @@ def log_val(self, x, out=None):
     def jax_forward(self):
         return self._forward_fn
 
+    @property
+    def jax_forward_nj(self):
+        return self._forward_fn_nj
```

Comment:
Do we really need to export this? 
I think we can keep it as a somewhat-private thing used inside of netket.

Reply:
I would actually need it to do the jvp/vjp 's in https://github.com/netket/netket/pull/517
but I agree that its not optimal.
Maybe its better to just just access the private member there?

Reply:
yep.
I'd say let's just not make it part of the public front facing API. 
Keep the field `_forward_fn_nj` but remove the `jax_forward_nj` property.